### ISSUE TITLE
Allow users to control display of the Project Tree tab in its dock

### ISSUE THREAD

**Issue by calebmeyer** [matthewlucock/pure-atom#3]
## Expected behavior
Here's atom in safe mode:
![image](https://user-images.githubusercontent.com/3342777/48087097-08116f00-e1c4-11e8-926e-b600a21ec51c.png)

## Actual behaviour (please include screenshots if appropriate)
Here's pure UI
![image](https://user-images.githubusercontent.com/3342777/48087089-02b42480-e1c4-11e8-836c-7e2ba8ad5036.png)

## Steps to reproduce the problem

  1. Open up the git pane
  1. Move it to the left side dock
  1. Now tree view is only accessible via keyboard

## Extra details (if applicable)
I really like having tree view and git in the same dock, so I can open/close them both at once. This package hides the tab for the tree view.

I don't expect this as the default, just something I'd like to have as an option: Maybe "Show Project Tab"?

## Specifications

  - Pure version: 1.0.3
  - Atom version: 1.32.1
  - Platform: Windows 10
  - Syntax theme: Atom Panda
  - Active community packages: 
<details>
  <summary>debug info with package info</summary>

```json
{
  "atom": "1.32.1",
  "platform": "win32",
  "release": "10.0.17134",
  "activeCommunityPackages": [
    "atom-panda-syntax: 0.18.0",
    "pure-ui: 1.0.3",
    "auto-detect-indentation: 1.3.0",
    "auto-update-plus: 0.5.8",
    "busy-signal: 1.4.3",
    "color-picker: 2.3.0",
    "file-icons: 2.1.26",
    "git-plus: 8.1.0",
    "minimap: 4.29.9",
    "multi-wrap-guide: 1.1.6",
    "nyan-indent: 0.9.8",
    "open-recent: 5.0.0",
    "rainbow-csv: 0.2.2",
    "ruby-block-converter: 4.1.1",
    "set-syntax: 0.4.0",
    "split-diff: 1.5.3",
    "symbol-gen: 1.3.1",
    "sync-settings: 0.8.6",
    "teletype: 0.13.3",
    "title-bar-replacer: 1.8.1",
    "vim-mode-plus: 1.36.0",
    "vim-mode-plus-keymaps-for-surround: 0.2.1",
    "django-templates: 0.5.0",
    "language-csv: 1.1.2",
    "language-aspx: 0.5.0",
    "language-groovy: 0.7.0",
    "language-rust: 0.4.12",
    "language-fish-shell: 1.1.0",
    "language-haml: 0.31.1",
    "language-pipfile: 0.2.0",
    "language-diff: 0.7.0",
    "react: 0.18.0",
    "language-powershell: 5.0.0",
    "language-docker: 1.1.8",
    "platformio-ide-terminal: 2.8.4",
    "gist-it: 0.9.2"
  ]
}
```

</details>

**Comment by matthewlucock** [comment]
I did deliberately hide the project tab for aesthetic reasons, but I definitely did not intend this side-effect.

Would a better solution be to just show the project tab if there are other tabs in the same dock, and hide it otherwise? I can do that as a one-line fix, and it would still achieve the aesthetic goal without , you know, breaking the tree view.

**Comment by calebmeyer** [comment]
That would be 100% better. I thought it would be impossible, so I didn't ask for it.

**Comment by matthewlucock** [comment]
Anything is possible when you believe 😉 But no, it was actually remarkably easy. [This is literally it](https://github.com/matthewlucock/pure-atom/commit/3a8b9be6ec172d9036b85de8049b80f542adf065). I've just published 1.0.4 with that fix. Update Pure and let me know if you're still having problems.

**Comment by calebmeyer** [comment]
Today I learned there's a :only-of-type pseudo selector. CSS just keeps getting better. Thanks for the super fast turnaround time. Happy to report this is fixed:
![image](https://user-images.githubusercontent.com/3342777/48149636-3f465580-e282-11e8-86c7-4b4265cdca10.png)
